Test/JsonDiffTest.cs

```diff
@@ -0,0 +1,34 @@
+using c_sharp_json_diff;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Test
+{
+    public class JsonDiffTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void WhenJsonDiffOnTwoStringsThenCorrectValueReturned()
+        {
+            string left = @"{
+                              'number': 10,
+                              'sommething': 20
+                             }";
+            string right = @"{
+                              'number': 5,
+                              'sommething': 10
+                             }";
+            JObject diffObject = JsonDiff.NumericDiff(left, right);
+            JArray array = (JArray)diffObject.Value<JToken>("number");
+            Assert.IsNotNull(array[2]);
```

Comment:
Instead of these 3 Assert statements, you could just check with ``Assert.equals`` I think that the array is exactly as expected: ``[5, 10, 51``. This will implicitly check everything you're already checking and more